vizro-core/CHANGELOG.md

```diff
@@ -11,6 +11,26 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
 
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.1.44'></a>
+
+# 0.1.44 — 2025-07-29
+
+## Added
+
+- Enable `show_select_all` argument in `vm.Checklist`, which enables users to remove 'Select All' option. ([#1283](https://github.com/mckinsey/vizro/pull/1283))
```

Comment:
```suggestion
- Add `show_select_all` argument to `vm.Checklist` so that users can remove 'Select All' option. ([#1283](https://github.com/mckinsey/vizro/pull/1283))
```